osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -63,8 +70,11 @@ protected override void Update()
         {
             base.Update();
 
+            float adjustedAlpha = ExponentialAlphaAdjust ? (float)Math.Pow(((Color4)DrawInfo.Colour.Colour).A, 3) : 1;

```

Comment:
Also, this is cubic, not exponential. So effectively, this just fades out children way faster than the parent to "hide" any discrepancies.